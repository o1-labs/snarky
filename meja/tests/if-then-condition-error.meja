let x = {
  if (15i) {
    1i;
  } else {
    2i;
  };
};
