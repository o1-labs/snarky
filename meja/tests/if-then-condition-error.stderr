File "tests/if-then-condition-error.meja", line 2, characters 8-9:
Error: Syntax error: ) expected
